### ISSUE TITLE
New game functionality

### ISSUE THREAD

**Issue by DeaconDesperado** [joeyespo/py-impactjs#4]
Would be awesome to have this.  I envision something that would build the skeleton directory structure in the filesystem and deploy default files.  Wouldn't need to initialize source control or anything, just something minimalistic.

The initial problem seems to be permissions.  Since Weltmeister assumes the server is on port 80, and port 80 is typically a privileged port requiring higher access level, is it fair to assume the server will have the proper permissions to write to the filesystem?  What are your feelings on this?

**Comment by joeyespo** [comment]
I think it's ok to assume write access. Flask apps usually allow this where necessary. There's even a [Flask-Upload](http://pythonhosted.org/Flask-Uploads/) for the common upload-file-to-disk use case.

Worst case, you don't get "new game" functionality if you're in a read-only environment. Which makes sense.

**Comment by joeyespo** [comment]
:+1: on this idea btw. It was an early intention of mine to experiment with something like this, but I moved on to other projects before I got around to it.

**Comment by DeaconDesperado** [comment]
@joeyespo I agree, we can assume write access and merely reject with a suitable error message if it is unavailable.

I feel you on the time constraints... that's what's nice about open source and the ability to share work : )

**Comment by joeyespo** [comment]
=)
